index.js

```diff
@@ -0,0 +1,299 @@
+        setTimeout(function(){calculator.querySelector('#six').classList
+        .remove('calculator-button-active')}, 100)
+        break
+      case '7':
+        calculator.querySelector('#seven').classList
+        .add('calculator-button-active')
+        setTimeout(function(){calculator.querySelector('#seven').classList
+        .remove('calculator-button-active')}, 100)
+        break
+      case '8':
+        calculator.querySelector('#eight').classList
+        .add('calculator-button-active')
+        setTimeout(function(){calculator.querySelector('#eight').classList
+        .remove('calculator-button-active')}, 100)
+        break
+      case '9':
+        calculator.querySelector('#nine').classList
+        .add('calculator-button-active')
+        setTimeout(function(){calculator.querySelector('#nine').classList
+        .remove('calculator-button-active')}, 100)
+        break
+      case '+':
+        calculator.querySelector('#add').classList
+        .add('calculator-button-right-active')
+        setTimeout(function(){calculator.querySelector('#add').classList
+        .remove('calculator-button-right-active')}, 100)
+        break
+      case '-':
+        calculator.querySelector('#subtract').classList
+        .add('calculator-button-right-active')
+        setTimeout(function(){calculator.querySelector('#subtract').classList
+        .remove('calculator-button-right-active')}, 100)
+        break
+      case '*':
+        calculator.querySelector('#multiply').classList
+        .add('calculator-button-right-active')
+        setTimeout(function(){calculator.querySelector('#multiply').classList
+        .remove('calculator-button-right-active')}, 100)
+        break
+      case '/':
+        calculator.querySelector('#divide').classList
+        .add('calculator-button-right-active')
+        setTimeout(function(){calculator.querySelector('#divide').classList
+        .remove('calculator-button-right-active')}, 100)
+        break
+      case 'Clear':
+        calculator.querySelector('#AC').classList
+        .add('calculator-button-clear-active')
+        setTimeout(function(){calculator.querySelector('#AC').classList
+        .remove('calculator-button-clear-active')}, 100)
+        break
+      case '.':
+        calculator.querySelector('#decimal').classList
+        .add('calculator-button-bottom-right-active')
+        setTimeout(function(){calculator.querySelector('#decimal').classList
+        .remove('calculator-button-bottom-right-active')}, 100)
+        break
+      case '=':
+        calculator.querySelector('#equals').classList
+        .add('calculator-button-bottom-right-right-active')
+        setTimeout(function(){calculator.querySelector('#equals').classList
+        .remove('calculator-button-bottom-right-right-active')}, 100)
+        break
+    }
+  }
+
+  document.querySelectorAll('.calculator-body').forEach(function(element){
+    var kal = new Calculator(element)
+    kal.addTheListeners()
```

Comment:
you do not need to call `.addTheListeners` here. Just do this work in your constructor